lib/db/schema/patch-083-084.sql

```diff
@@ -0,0 +1,57 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('83');
+
+-- Since we are altering the size of the `recoveryKeyId` column
+-- we need to make sure that the column is empty before it can be applied
+DELETE from recoveryKeys;
+
+ALTER TABLE recoveryKeys MODIFY COLUMN recoveryKeyId BINARY(16);
+
+CREATE PROCEDURE `createRecoveryKey_2` (
+  IN `uidArg` BINARY(16),
+  IN `recoveryKeyIdArg` BINARY(16),
+  IN `recoveryDataArg` TEXT
+)
+BEGIN
+
+  DECLARE EXIT HANDLER FOR SQLEXCEPTION
+  BEGIN
+    ROLLBACK;
+    RESIGNAL;
+  END;
+
+  START TRANSACTION;
+
+  SET @accountCount = 0;
+
+  -- Signal error if no user found
+  SELECT COUNT(*) INTO @accountCount FROM `accounts` WHERE `uid` = `uidArg`;
+  IF @accountCount = 0 THEN
+    SIGNAL SQLSTATE '45000' SET MYSQL_ERRNO = 1643, MESSAGE_TEXT = 'Can not create recovery key for unknown user.';
+  END IF;
```

Comment:
I didn't notice this before, but I wonder if it's really necessary.  I don't recall any of our other stored procedures doing this kind of check, do they?

Anyway, it exists in the previous version of this stored procedure so I'm fine with leaving it in, it just stood out as potentially inconsistent with other tables/procedures.